### PR TITLE
K8SPXC-264 Get openshift ca

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

File: e2e-tests/functions [hunk 1]
```diff
@@ -725,6 +725,9 @@ EOF
     kubectl_bin get csr ${CSR_NAME} -o jsonpath='{.status.certificate}' > ${tmp_dir}/serverCert
     openssl base64 -in ${tmp_dir}/serverCert -d -A -out ${tmp_dir}/vault.crt
     kubectl_bin config view --raw --minify --flatten -o jsonpath='{.clusters[].cluster.certificate-authority-data}' | base64 -d > ${tmp_dir}/vault.ca
+    if [[ "${OPENSHIFT}" == 1 ]]; then
+        kubectl_bin -n default get secrets  | grep default |grep service-account-token | head -n 1 | awk {'print $1'} | xargs -I {} kubectl_bin -n default get secret {} -o jsonpath='{.data.ca\.crt}' | base64 -d > ${tmp_dir}/vault.ca
```
Comment (delgod) [inline e2e-tests/functions:729]:
```suggestion
        kubectl_bin -n default get secrets \
            | grep default \
            | grep service-account-token \
            | head -n 1 \
            | awk {'print $1'} \
            | xargs -I {} kubectl_bin -n default get secret {} -o jsonpath='{.data.ca\.crt}' \
            | base64 -d > ${tmp_dir}/vault.ca
```

Comment (cap1984) [reply e2e-tests/functions]: done

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| storage | passed |
| big-data | passed |
| recreate | passed |
| scaling | passed |
| limits | passed |
| self-healing | passed |
| monitoring | passed |
| scaling-proxysql | passed |
| restore-to-encrypted-cluster | failed |
| upgrade | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| self-healing-advanced | failed |
| affinity | passed |
| upgrade-consistency | passed |
| operator-self-healing | passed |
| security-context | passed |
| one-pod | passed |
| scheduled-backup | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| big-data | passed |
| scaling | passed |
| recreate | passed |
| init-deploy | failed |
| limits | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| restore-to-encrypted-cluster | failed |
| upgrade | passed |
| monitoring | passed |
| demand-backup | passed |
| self-healing-advanced | passed |
| monitoring-2-0 | passed |
| upgrade-consistency | passed |
| security-context | passed |
| operator-self-healing | passed |
| affinity | passed |
| one-pod | passed |
| scheduled-backup | failed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| scaling | passed |
| recreate | passed |
| init-deploy | failed |
| storage | passed |
| limits | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| restore-to-encrypted-cluster | failed |
| upgrade | passed |
| monitoring | passed |
| demand-backup | passed |
| upgrade-consistency | passed |
| self-healing-advanced | failed |
| monitoring-2-0 | passed |
| security-context | passed |
| affinity | passed |
| operator-self-healing | passed |
| scheduled-backup | failed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| big-data | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| recreate | passed |
| restore-to-encrypted-cluster | failed |
| init-deploy | failed |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| big-data | passed |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| init-deploy | failed |
| security-context | passed |
| recreate | passed |
| restore-to-encrypted-cluster | failed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| storage | passed |
| init-deploy | failed |
| big-data | passed |
| restore-to-encrypted-cluster | failed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | passed |
| storage | passed |
| big-data | passed |
| recreate | passed |
| limits | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| restore-to-encrypted-cluster | passed |
| upgrade | passed |
| monitoring | passed |
| self-healing-advanced | passed |
| upgrade-consistency | passed |
| security-context | passed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| demand-backup | passed |
| affinity | passed |
| one-pod | passed |
| scheduled-backup | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | passed |

**Pull request merged**